docs/source/procedures/creating-releases.rst

```diff
@@ -0,0 +1,50 @@
+The Augur Release Process
+=========================
+
+The first step to releasing any changes is to have changes in the first place.
+Augur's `CONTRIBUTING.md <https://github.com/chaoss/augur/blob/main/CONTRIBUTING.md>`__ file contains all the information that is needed to get started with topics like reporting issues, contributing code, and understanding the code review process.
+
+This document outlines how these changes end up in an Augur release after they are merged into the `main` branch.
+
+
+
+Release workflow:
+Starting after version 0.89.3, Augur follows a workflow similar to those you may already be familiar with (such as github flow and git flow). The Augur workflow has two long-lived branches, `main` and `release` and is designed such that changes only flow in one direction - from main into release.
+
+Branches
+
+`main` 
+The `main` branch is the primary development branch that is the target for all new pull requests. At any given point in time, this branch represents the best approximation of what the next upcoming release will look like. Since this is the active development branch, changes happen more frequently and this branch should be considered to be less stable than the `release` branch due to the possibility of breaking changes being made (and potentially reverted) between releases. It is not recommended for production deployment and is primarily intended for use by Augur contributors running their own copies against test data for development purposes.
+
+`release`
+The `release` branch is where all augur versions (after 0.89.3) are tagged. Each commit on this branch represents either a hotfix to the prior release or a new major or minor version.
+
+Currently, Augur only officially supports the last-released version represented by the latest commit on the `release` branch. 
+
+.. note::
+
+    If future needs require supporting multiple Augur versions concurrently, individual numbered release branches may be made from this central `release`` branch to allow any hotfixes to be applied to each supported version independently of the others. 
+
+
+The Release Process
+
+When the next release is set to be cut, there are some preparation steps need to take place first, these include:
```

Comment:
We are discussing how this would work. We think : 
- Everything gets merged into main
- When we do a release it all gets merged into release
- If there are hot fixes into main, those may need to be cherry picked into release. 


I think we have this understanding correct.